dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/TrackerBundle.java

```diff
@@ -129,6 +130,8 @@ public class TrackerBundle {
   private Map<TrackerType, Map<String, TrackerImportStrategy>> resolvedStrategyMap =
       initStrategyMap();
 
+  @Builder.Default @JsonIgnore private Set<String> updatedTeis = new HashSet<>();
```

Comment:
its odd that this is happening 😬 we did apply the formatting on the entire codebase, so why does it do that now for you?

Reply:
Could it be because of this?
https://github.com/dhis2/dhis2-core/pull/14624

Reply:
I don't think so because if I run `mvn spotless:apply -f ./dhis-2/pom.xml` there are no changes. Are you formatting via the IDE? If so how (is it configured)?

Reply:
I was formatting the whole file in the `Action on Save `. Changing that to `Change lines` solved the issue